src/screens/Swap/Form/SelectAccount/BadSelectionModal.js

```diff
@@ -27,7 +27,10 @@ const BadSelectionModal = ({
 }) => {
   const { navigate } = useNavigation();
   const openManagerForApp = useCallback(() => {
-    navigate(ScreenName.Manager, {});
+    navigate(NavigatorName.Manager, {
+      screen: ScreenName.Manager,
+      params: { searchParams: "Exchange" },
```

Comment:
Here it would be something like `mainCurrency.managerAppName` where `mainCurrency` is currency or currency.parentCurrency depending if it's a token.